include/SDL3/SDL_gpu.h

```diff
@@ -1141,12 +1141,12 @@ typedef struct SDL_GPUTextureCreateInfo
 {
     SDL_GPUTextureType type;
     SDL_GPUTextureFormat format;
-    SDL_GPUTextureUsageFlags usageFlags;
+    SDL_GPUTextureUsageFlags usage_flags;
     Uint32 width;
     Uint32 height;
-    Uint32 layerCountOrDepth;
-    Uint32 levelCount;
-    SDL_GPUSampleCount sampleCount;
+    Uint32 layer_count_or_depth;
```

Comment:
layers? With field documentation?

Reply:
or num_layers?

Reply:
This one is tricky because it's one field that has two mutually-exclusive meanings, like a pseudo-union. If the texture type is 3D, then this represents depth. If the texture type is Array/Cube, then this represents layer count. Not sure how to succinctly explain that via the field name.

Reply:
What if you use an actual union?

Reply:
This might be my preference coming from doing a lot of interop but I find unions confusing and annoying to bind to non-C languages and I prefer to avoid them in public APIs when possible.

Reply:
It's a weird corner case, "layer_count_or_depth" is fine in this case, imho, unless you want to split it into two separate fields and check the right one depending on texture type, but that's confusing in a different way. I say leave it as-is.